tests/testthat/test-setup.R

```diff
@@ -737,7 +737,7 @@ test_that("reports and plots are produced", {
   # Not testing right now
   #expect_no_error(
     #p <-
-      #traits.build::plot_trait_distribution_beeswarm(
+      #austraits::plot_trait_distribution_beeswarm(
```

Comment:
We should just delete these commented lines, as the function is tested in `austraits1